### ISSUE TITLE
Kindle stylesheet opinion

### ISSUE THREAD

**Issue by dunxd** [mikespub-org/seblucas-cops#56]
I noticed that the Kindle style for the default theme includes code insert a credit at the end of the page:

https://github.com/mikespub-org/seblucas-cops/blob/111d4e9da164d1e1f15cf947b5ddf454843398b0/templates/default/styles/style-kindle.css#L488-L513

I'm all for giving credit where credit is due, but it is more prominent than credits to seblucas or anyone else involved, and using unicode characters to obscure the text in source doesn't sit right with me.

Have I just got up on the wrong side of the bed this morning?

**Comment by dunxd** [comment]
However it has got me thinking about using unicode characters in place of icons for my kindle oriented template.

**Comment by mikespub** [comment]
Heh - guess credits aren't all bad then :-)

I just included the style from https://github.com/Joduai/cops-kindle without any changes when I came across it, as I have no Kindle to test this on and I'm probably not the best person to judge front-end styles. Maybe you and/or @Joduai can improve on it for future releases?

**Comment by dunxd** [comment]
See #59

**Comment by Joduai** [comment]
@dunxd @mikespub 
sorrz for "a bit" late reply, did not get a single notice after mentioning me

everything clarifies this old issue: https://github.com/seblucas/cops/issues/422
I just went from simple "patch" to have readable online library in kindle's experimental browser, to stand alone "theme".
I didn't even know there's a COPS fork until recently I resurrected my old online library, and checked if there's a new player on OPDS ground. The more, it even contains this css theme that years ago used not more than 3 people including me.
 
Nonetheless feel free to edit the css as you wish if it's still a matter ;)

After some time I realized that on desktop browser search input is animated, but didn't have the hearth to go back and fix its behavior.
